src/App.js

```diff
@@ -14,35 +13,53 @@ class App extends Component {
     super(props);
     this.state = {
       errorMessage: null,
+      transactionState: {
+        transactions: [],
+        sorted: {},
+      },
     };
   }
 
   setErrorMessage = (message) => this.setState({ errorMessage: message });
 
+  addTransactionToTransactionState = (transaction) =>
```

Comment:
from what i saw in react tutorial, a convention for something like this should be called `handleAddTransaction()`.

the example was for `onClick()` and `handleClick()`, but i think it could make sense here too

Reply:
The tutorial I saw liked using the "set" prefix for when it was setting a piece of state to a new value, so this seems to be a preference thing.

That said, I am fine using handleAddTransaction so I will update it.